qiskit_braket_provider/providers/adapter.py

```diff
@@ -197,6 +201,23 @@
     ),
     "gphase": qiskit_gates.GlobalPhaseGate(Parameter("theta")),
     "measure": qiskit_gates.Measure(),
+    "kraus": qiskit_qi.Kraus([[[1, 0], [0, 0]], [[0, 0], [0, 1]]]).to_instruction(),
+}
+
+_BRAKET_SUPPORTED_ERRORS = {
```

Comment:
It appears that you only use the keys of this in the source code. `_BRAKET_SUPPORTED_ERRORS` can just be a list of names. The values are not needed i the source code. 

The values are only used in tests. You can add then as test data.

Reply:
nit: let's use a consistent name ("_BRAKET_SUPPORTED_NOISES").